libs/community/langchain_community/agent_toolkits/sql/base.py

```diff
@@ -235,5 +237,6 @@ def create_sql_agent(
         max_iterations=max_iterations,
         max_execution_time=max_execution_time,
         early_stopping_method=early_stopping_method,
+        handle_parsing_errors=True,
```

Comment:
this will change behavior for users. why add it? is it related to the fix?

Reply:
reverted